### ISSUE TITLE
leaflet maps are scroll traps

### ISSUE THREAD

**Issue by thatbudakguy** [Princeton-CDH/mep-django#639]
# testing notes

when you navigate to a page with a map (currently just the map on member detail pages), the map should no longer scroll using your scroll wheel. You should be able to scroll all the way down the page without interacting with the map. the map should still zoom in/out using its +/- zoom controls.

# dev notes
from [john's review](https://docs.google.com/spreadsheets/d/1N-oBy9Mp91TEdXEi6rjCMNsbJNacxMh6CEGTHpCRfws/edit?usp=sharing):
> The map on the member lists automatically scrolls as a user scrolls over it. This can "trap" the scroll wheel, which can be quite disorienting for screen magnifier users.

this is potentially major, since it affects both the member detail map and search map (#201), and could also allow us to make the maps full-width/height more easily.

**Comment by rlskoeser** [comment]
Can confirm this is working as documented on the test site, but kind of a bummer not to be able to easily zoom in. @thatbudakguy is there some way to enable scroll zooming when you select the map (in future)?

**Comment by rlskoeser** [comment]
I'll go ahead and close since this is working as expected and addresses the accessibility problem.
